config/store_manage.yml

```diff
@@ -1,8 +1,8 @@
 production:
-  ffmpeg_path: '/usr/local/bin/ffmpeg'
+  ffmpeg_path: '/app/vendor/ffmpeg/ffmpeg'
 
 development:
   ffmpeg_path: '/usr/local/bin/ffmpeg'
 
 test:
-  ffmpeg_path: '/usr/local/bin/ffmpeg'
\ No newline at end of file
+  ffmpeg_path: '/usr/local/bin/ffmpeg'
```

Comment:
I think we could move this paths to `ENV` variables

Reply:
Agreed